test/Suites/Plutus/Model/Script/V1/Test/Counter.hs

```diff
@@ -45,7 +45,7 @@ initCounterTest = do
   counterVal <- valueAt counterScript
   counterUtxos <- utxoAt counterScript
   let [(counterRef, counterOut)] = counterUtxos
-  mDat <- datumAt @CounterDatum counterRef
+  mDat <- datumAt @_ @CounterDatum counterRef
```

Comment:
Use Monad parameter as second argument in this function and everywhere else
to avoid @_. As script type changes more often than monad type